modules/graphql_voyager/assets/src/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import { Voyager } from 'graphql-voyager';
+import fetch from 'isomorphic-fetch';
+import Drupal from 'drupal';
+import jQuery from 'jquery';
+
+/**
+ * Behavior for rendering the GraphQL Voyager interface.
+ */
+Drupal.behaviors.graphQLRenderVoyager = {
+  attach: (context, settings) => {
+    const container = jQuery('#graphql-voyager', context).once('graphql-voyager')[0] || undefined;
+    const INTROSPECTION_URL = settings.graphQLRequestUrl || `${window.location.origin}/graphql`;
+
+    if (typeof container === 'undefined') {
+      return;
+    }
+
+    class VoyagerWrapper extends Component {
+      constructor() {
+        super();
```

Comment:
This comment is meaningless once you convert this to a functional stateless component... However, if you do not do anything in the constructor other than calling super(), you can omit it entirely.